node/pkg/p2p/p2p.go

```diff
@@ -280,14 +280,14 @@ func Run(
 			ourAddr := ethcrypto.PubkeyToAddress(gk.PublicKey)
 
 			ctr := int64(0)
-			tick := time.NewTicker(15 * time.Second)
-			defer tick.Stop()
```

Comment:
Shouldn't we keep the defer or put a tick.Stop() in the ctx.Done() case below?

Reply:
good catch, this should've been changed to time.Timer. The reason to switch to `time.Timer` instead of `time.Ticker` was so that we can do the first iteration after one millisecond instead of 15 seconds, such that guardian immediately send out a heartbeat and the tests complete faster.